.github/workflows/tests.yml

```diff
@@ -20,19 +21,50 @@ jobs:
       group: ${{ github.ref == 'refs/heads/main' && format('unit-tests-main-{0}', github.sha) || github.ref == 'refs/heads/develop' && format('unit-tests-develop-{0}', github.sha) || format('unit-tests-{0}', github.ref) }}
       cancel-in-progress: true
     steps:
-      - uses: actions/checkout@v3
-      - name: Run tests and generate kover report
-        run: ./gradlew koverMergedReport $CI_GRADLE_ARG_PROPERTIES
+      - name: ⏬ Checkout with LFS
+        uses: actions/checkout@v3
+        with:
+          lfs: 'true'
+
+      - name: ⚙️ Run unit & screenshot tests, generate kover report
+        run: ./gradlew koverMergedReport $CI_GRADLE_ARG_PROPERTIES -Pci-build=true
+
+      - name: 📈 Verify coverage
+        run: ./gradlew koverMergedVerify $CI_GRADLE_ARG_PROPERTIES -Pci-build=true
```

Comment:
`koverMergedReport` and `koverMergedVerify` will both run the tests IIRC. It's a shame there is not a single task to do both without running the test twice...

Reply:
But we just launched `verifyPaparazzi` from the previous `koverMergedReport`, so if I'm not mistaken it should be up to date and not run again. At least, I don't see any delay when running these, I'll double check just in case.